modules/analyzing-the-collector-pod-status.adoc

```diff
@@ -16,21 +16,24 @@ Examining the pod's most recent status is another easy way to determine the caus
 $ oc describe pod -n stackrox _<collector_pod_name>_ <1>
 ----
 +
-<1>  If you use Kubernetes, enter `kubectl` instead of `oc`. For 
-`_<collector_pod_name>_`, specify the name of your Collector pod, for example, `collector-vclg5`.
+--
+<1>  If you use Kubernetes, enter `kubectl` instead of `oc`. For `_<collector_pod_name>_`, specify the name of your Collector pod, for example, `collector-vclg5`.
+--
 +
 .Example output
 +
-[source,terminal]
+[source,text]
 ----
-[...]
+# ...
     Last State:     Terminated
       Reason:       Error
       Message:      No suitable kernel object downloaded <1>
       Exit Code:    1
       Started:      Fri, 21 Oct 2022 11:50:56 +0100
       Finished:     Fri, 21 Oct 2022 11:51:25 +0100
-[...]
+# ...
 ----
 +
-<1> In this example, you can see that Collector has failed to download a kernel driver.
\ No newline at end of file
+--
```

Comment:
Were you having problems with the code callouts not rendering properly in the past? 

In general, we don't enclose callouts in `--` unless there is some sort of formatting problem to solve, usually due to more complex markup around it.

For consistency, I would recommend dropping these unless there is a strong reason to do keep them.

Reply:
Yes, I had consecutive travis build failures due to these callouts. The strange part is when I ran Travis locally it did not return an error, but when I pushed the changes to remote the build failed. You can also see that I have made adjustments in several other modules as well. In all other cases where I used `--`, the build error was fixed.